nautobot_circuit_maintenance/handle_notifications/email_helper.py

```diff
@@ -39,17 +39,11 @@ class Config:
     def __post_init__(self):
         """Post init validation."""
         if self.user is None:
```

Comment:
I'm surprised that these checks are needed, given that `user`, `password`, and `imap_url` are explicitly declared above as `str` (and not `Optional[str]`). Doesn't Pydantic automatically reject initialization with None values for any of these parameters?

Reply:
I agree with you... the validation was slightly different in the past... and the inercia left them here